docs/src/assets/chainrules.scss

```diff
@@ -74,9 +86,8 @@ html:not(.theme--documenter-dark) body #documenter {
 
   code {
     color: inherit;
-
+		font-family: "JuliaMono";
```

Comment:
I think we need this for `pre` as well. (Should also probaby use spaces instead of tabs)

Reply:
OK!